app/_hub/kong-inc/exit-transformer/index.md

```diff
@@ -142,7 +190,7 @@ return function(status, body, headers)
 end
 ```
 
-* Modify the body and headers, even if none were provided
+#### Modify the body and headers regardless if provided {#mod-body-head}
```

Comment:
"Regardless if" is an awkward construction, not sure that's grammatically correct. I believe the proper construction is "regardless of if", which is even more awkward in this case - I'd avoid using "regardless" entirely to simplify. 

Struggling to come up with a good phrase here though. I actually prefer the original phrasing over the rephrase, but that might be a personal choice.

Reply:
stet